### ISSUE TITLE
tls wrapping

### ISSUE THREAD

**Issue by PurpleCroquet** [linode/docs#219]
After following the great guideline I could get email out. After uncommenting `smtpd_tls_wrappermode=yes `I got it working on port 465

this is a where it happened: `/etc/postfix/main.cf` 
```
 smtps     inet  n       -       -       -       -       smtpd
#  -o syslog_name=postfix/smtps
-o smtpd_tls_wrappermode=yes
#  -o smtpd_sasl_auth_enable=yes
#  -o smtpd_client_restrictions=permit_sasl_authenticated,reject
#  -o milter_macro_daemon_name=ORIGINATING
```

**Comment by alexfornuto** [comment]
Hey, can you please clarify which guide you're referring to?

**Comment by PurpleCroquet** [comment]
this one: https://www.linode.com/docs/email/postfix/email-with-postfix-dovecot-and-mysql

**Comment by tknarr** [comment]
In both cases I think the block of option lines below each entry ('submission' and 'smtps') should also be uncommented. The 'submission' port should only be for authenticated users to send through, and the 'smtps' port is deprecated and only needed for client programs that can't use the 'submission' port for some reason (eg. they don't support STARTTLS, they can't be configured to use that port number).

**Comment by tknarr** [comment]
Pull request #234 submitted to fix master.cf

**Comment by thetechstewart** [comment]
Fixed by PR #234.
